src/states/multispecifications/testData.js

```diff
@@ -0,0 +1,59 @@
+export const defaultValues = {
```

Comment:
In #82, I moved the dimension testDataFile to the mockdata folder. We should discuss how we handle it in the future, if we either have the test data in the tested store or on its own.